packages/wrangler/src/user/user.ts

```diff
@@ -361,23 +361,16 @@ const DefaultScopes = {
 		"See and change Cloudflare Pipelines configurations and data",
 	"secrets_store:write":
 		"See and change secrets + stores within the Secrets Store",
-} as const;
-
-const OptionalScopes = {
+	"containers:write": "Manage Workers Containers",
 	"cloudchamber:write": "Manage Cloudchamber",
 } as const;
 
-const AllScopes = {
-	...DefaultScopes,
-	...OptionalScopes,
-};
-
 /**
  * The possible keys for a Scope.
  *
  * "offline_access" is automatically included.
  */
-export type Scope = keyof typeof AllScopes;
+export type Scope = keyof typeof DefaultScopes;
```

Comment:
just double checking...so `"cloudchamber:write"` is intentionally excluded from this scopes list correct?